msteams-platform/task-modules-and-cards/cards/people-picker.md

```diff
@@ -0,0 +1,305 @@
+The following properties are additions to the `Input.ChoiceSet` schema to enable People Picker experience on the card:  
+
+#### Input.ChoiceSet control
+
+|Property |Type |Required |Description |
+|----|----|----|----|
+|**choices.data** |**Data.Query** |No |Enables dynamic auto complete for different user types, by fetching results from the dataset specified. |
+
+#### Data.Query
+
+|Property |Type |Required |Description|
+|--|--|--|--|
+|**dataset** |String |Yes |The type of data that must be fetched dynamically.|   
+
+#### dataset
+The following table provides predefined values as **dataset** for people picker:   
+
+|dataset|Search Scope
+|--|--|
+|**graph.microsoft.com/users** |Search all members across the organization.|
+|**graph.microsoft.com/users?scope=currentContext** |Search within the members of the current conversation, such as chat or channel in which the particular card is sent.|      
+
+> [!NOTE]
+> Currently, the search functionality for all the members across the organization is supported in 1:1 chats with bots, Adaptive Card task modules and tabs only. It is not supported in other 1:1 chats, group chats or channels.  
+
+### Example
+The code example for creating People Picker with organization search is as follows:
+
+```json 
+{
+	"type": "AdaptiveCard",
+	"body": [
+		{
+			"type": "TextBlock",
+			"size": "Medium",
+			"weight": "Bolder",
+			"text": "People Picker with Org search enabled"
+		},
+		{
+			"type": "Input.ChoiceSet",
+			"choices": [],
+			"choices.data": {
+				"type": "Data.Query",
+				"dataset": "graph.microsoft.com/users"
+			},
+			"id": "people-picker",
+			"isMultiSelect": true
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "Submit"
+		}
+	],
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"version": "1.2"
+}
+```  
+
+The following image illustrates People Picker in Adaptive Cards with organization search:
+
+![People Picker Org Search](../../assets/images/cards/peoplepicker-org-search.png)
+
+To enable search within a list of conversation members, use the appropriate dataset defined in the [dataset](#dataset) table. `isMultiSelect` property is used to enable the selection of multiple users  in the control. It's set to false by default and this setting allows you to select single user only.
+
+### Data Submission
+
+You can use `Action.Submit` or `Action.Execute` to submit selected data to your bot. The `invoke` payload received on your bot is a list of AAD IDs or the IDs provided in static list.
+In People Picker, when a user is selected in the control, the `AAD ID` of the user is the value sent back. The `AAD ID` is a string and uniquely identifies a user in the directory.
+
+The format of the value submitted to the bot depends on the value of the `isMultiSelect` property:
+
+|value of `isMultiSelect`|Format|
+|--|--|
+|false _(single select)_|<selected_AAD_ID>.|
+|true _(multi select)_|<selected_AAD_ID_1>,<selected_AAD_ID_2>,<selected_AAD_ID_3>.|  
```

Comment:
Please remove the period after <selected_AAD_ID_3>

this is how it should be:
|true _(multi select)_|<selected_AAD_ID_1>,<selected_AAD_ID_2>,<selected_AAD_ID_3>|

Reply:
@v-abirade can you make this change?

Reply:
removed